examples/blender/blender.py

```diff
@@ -75,9 +75,17 @@ async def worker(ctx: WorkContext, tasks):
             script.download_file(f"/golem/output/out{frame:04d}.png", output_file)
```

Comment:
or, alternatively, I'd just copy the example to something else - e.g. `examples/reputation-poc/` ... or something